spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -1032,16 +1038,24 @@
       end
     end
 
-    context "pediatric neurology brief collection" do
+    context "file belongs to pediatric neurology brief collection" do
       before do
-        make_collection(user, title: "Pediatric Neurology Briefs: Volume 05, Issue 07", id: "pnb-5-7", member_ids: [generic_file_pnb.id])
-
         collection_store.build_collection_store_data
         collection_store.build_paths_for_collection_store
       end
 
-      it "returns the correct community string" do
-        expect(pnb_collection.send(:dh_collection_to_prism_community_collection)).to eq(expected_pnb_community_string)
+      describe "it returns the correct community string" do
+        context "when issue number in collection id" do
+          it "returns the correct community string for a pnb generic file" do
+            expect(generic_file_converter_pnb.send(:dh_collection_to_prism_community_collection)).to eq(expected_pnb_community_string)
+          end
+        end
+
+        context "when issue number not in collection id" do
+          it "returns the correct community string for a" do
```

Comment:
minor: 

- "for a"...?

- Also `describe "it returns the correct community string"` and then `it "returns the correct community string..."` is repeated. Is it possible to remove a layer of context/describe?

Reply:
Good catch. I've changed the strings to a be more descriptive. 

I think we could remove the string from the spec itself, so `it "test description do` could be replaced with `it do` , but I'm going to leave the strings in for now since they can be helpful for identifying failures 

```
    context "for a pnb generic file" do
        context "that belongs to a collection with an issue number in the id" do
            it "returns the correct community string" do
        context "that belongs to a collection without an issue number in the collection id" do
            it "returns the correct community string" do
```